class-block-unit-test.php

```diff
@@ -210,6 +210,9 @@ public function upgrade_completed( $upgrader_object, $options ) {
 	 */
 	public function content() {
 
+		$layout_settings = gutenberg_get_global_settings(array( 'layout' ));
```

Comment:
this function has been renamed to wp_get_global_settings - ref https://github.com/WordPress/gutenberg/pull/36907

Reply:
done in https://github.com/godaddy-wordpress/block-unit-test/pull/11/commits/efc29a8984f6533e501f7e0c9d2f29042d0b5335